src/main/java/pixelperfect/route/AsteroidNode.java

```diff
@@ -0,0 +1,24 @@
+package pixelperfect.route;
+
+/**
+ * An asteroid field Node which belongs in a Route. The asteroid field will be a dangerous location
+ * to fly in.
+ * 
+ * @author Floris Doolaard
+ *
+ */
+public class AsteroidNode extends RouteNode {
+
+  /**
+   * The constructor for the AsteroidNode.
+   * 
+   * @param summary
+   *          , a summary of the AsteroidNode.
+   * @param description
+   *          , a description of the AsteroidNode.
+   */
+  public AsteroidNode(String summary, String description) {
+    super(summary, description);
```

Comment:
Indeed, I have though of this as well.

I found a counter example. Let's say you have multiple Routes or RouteNodes. If you want one AsteroidField to be large and complex and the other one to be small and easy you might want to have different descriptions/summaries.